### PR TITLE
Shutdown behaviours implementation

### PR DESCRIPTION
**Pull request by AlexisMontagne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,19 @@ GOPATH=`pwd`/Godeps/_workspace go build -o etcdenv .
 | ------ | ------- | ----------- |
 | `server`, `s` | http://127.0.0.1:4001 | Location of the etcd server |
 | `namespace`, `n`| /environments/production | Etcd directory where the environment variables are fetched. You can watch multiple namespaces by using a comma-separated list (/environments/production,/environments/global) |
-| `r` | false | Not restart the command when a key watch change |
+| `shutdownBehavour`, `b` | keepalive | The strategy applied when the process exit, further information into the next paragraph |
```
Comment (jlevesy) [inline README.md]: "Strategy to apply when process exits" ?

File: README.md [hunk 2]
```diff
@@ -35,9 +35,19 @@ GOPATH=`pwd`/Godeps/_workspace go build -o etcdenv .
...
 | `watched`, `w` | `""` | A comma-separated list of environment variables triggering the command restart when they change |
 
+
+### Shutdown strategies
+
+* `restart`: When `etcdenv` detects that the process is no longer alive
```
Comment (jlevesy) [inline README.md]: run is an irregular verb.  Runned -> run.

Comment (jlevesy) [inline README.md]: Rerun command on  child process death ?

File: README.md [hunk 3]
```diff
@@ -35,9 +35,19 @@ GOPATH=`pwd`/Godeps/_workspace go build -o etcdenv .
...
+  a new instance of the command is runned
+* `exit`:  The `etcdenv` process exits with the same exit status as the
+  wrapped process's
```
Comment (jlevesy) [inline README.md:46]: child process ?

**Comment by jlevesy** [comment]
Nice feature ! 
I don't think I'm relevant to do the code review. But I commented some stuff on the README.
Summon @phndiaye  ;)

File: README.md [hunk 4]
```diff
@@ -35,9 +35,18 @@ GOPATH=`pwd`/Godeps/_workspace go build -o etcdenv .
 | ------ | ------- | ----------- |
 | `server`, `s` | http://127.0.0.1:4001 | Location of the etcd server |
 | `namespace`, `n`| /environments/production | Etcd directory where the environment variables are fetched. You can watch multiple namespaces by using a comma-separated list (/environments/production,/environments/global) |
-| `r` | false | Not restart the command when a key watch change |
+| `shutdownBehavour`, `b` | keepalive | Strategy to apply when the process exit, further information into the next paragraph |
```
Comment (phndiaye) [inline README.md]: camelCase in command options looks weird to me ... is not `shutdown-behavior` nicer ? (plus the typo — Behavour)

**Comment by phndiaye** [comment]
LGTM (aside of the typo in the README)

**Pull request merged**